2-mandatory/3-financial-times.js

```diff
@@ -14,7 +14,11 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+  let result = allArticleTitles[0]
+  for (let i = 0; i < allArticleTitles.length; i++) {
+    if (allArticleTitles[i].length < result.length) result = allArticleTitles[i]
+  }
+  return result
```

Comment:
As we talked about on our call, please update this for fewest words not fewest letters

Reply:
Thanks @DiegoJunior0 ! I've sent the solution in slack for team.

    if (allArticleTitles[i].split(' ').length < result.split(' ').length)

Reply:
Thanks @DiegoJunior0 ! I've sent it on Slack for team.

    if (allArticleTitles[i].split(' ').length < result.split(' ').length)

Reply:
I'll change it in project as well.